experimental/CHANGELOG.md

```diff
@@ -120,9 +120,9 @@ All notable changes to experimental packages in this project will be documented
 * fix(otlp-exporter-base)!: remove unload event from OTLPExporterBrowserBase [#4438](https://github.com/open-telemetry/opentelemetry-js/pull/4438) @eldavojohn
   * Reason: The 'unload' event prevents sites from taking advantage of Google's [backward/forward cache](https://web.dev/articles/bfcache#never_use_the_unload_event) and will be [deprecated](https://developer.chrome.com/articles/deprecating-unload/).  It is now up to the consuming site to implement these shutdown events.
   * This breaking change affects users under this scenario:
-    1. A user extends the exporter and overrides the shutdown function, and does something which is usually called by the unload listener
-    2. We remove the unload event listener
-    3. That user's overridden shutdown function no longer gets called
+    * A user extends the exporter and overrides the shutdown function, and does something which is usually called by the unload listener
+    * We remove the unload event listener
+    * That user's overridden shutdown function no longer gets called
```

Comment:
Was this change intentional?

Reply:
Yes, now that this file participates in the CHANGLELOG linting in CI, the linter emitted errors about it

Reply:
Hrm. That feels like a bug in markdown-lint then. Is an ordered list under an unordered list not allowed?

Reply:
I was trying to find proper guidance and struggling a bit. My thought is generally to follow what HTML allows, which is that a list can be ordered or unordered, not mixed. Looks like [GitHub Flavored Markdown spec](https://github.github.com/gfm/) suggests:

> A [list](https://github.github.com/gfm/#list) is a sequence of one or more list items [of the same type](https://github.github.com/gfm/#of-the-same-type). The list items may be separated by any number of blank lines.

So I think technically ordered sub-items should not exist in an unordered list.

Reply:
> My thought is generally to follow what HTML allows, which is that a list can be ordered or unordered, not mixed.

IIUC, this is about the list item in an unordered list containing an ordered list.
Like this: https://gist.github.com/trentm/ab0df8f9ded43ec474d5f06b4db3ab6d

Markdown and HTML allow this.

Reply:
Playing with markdown-lint separately (`npm install markdown-lint`, `./node_modules/.bin/markdown-lint foo.md`):

It is okay with this:

```md
-   one

-   two

-   three

    1. un
    1. deux
    1. trois
```

It doesn't like the sane bullet numbering in this:

```md
-   one

-   two

-   three

    1. un
    2. deux
    3. trois
```

```
% ./node_modules/.bin/markdown-lint foo.md
<stdin>
  8:5-8:12  warning  Marker should be `1`, was `2`  ordered-list-marker-value  remark-lint
  9:5-9:13  warning  Marker should be `1`, was `3`  ordered-list-marker-value  remark-lint

⚠ 2 warnings
```

It is also over picky (IMHO) with "tight" lists (as GFM calls them, https://github.github.com/gfm/#tight):

```md
- one
- two
- three
    1. un
    1. deux
    1. trois
```

```
% ./node_modules/.bin/markdown-lint foo.md
<stdin>
  1:6-2:1  warning  Missing new line after list item  list-item-spacing  remark-lint
  2:6-3:1  warning  Missing new line after list item  list-item-spacing  remark-lint

⚠ 2 warnings
```

@blumamir If you recall, was the markdown-lint error about the numbering of bullets, do you recall?

```
Marker should be `1`, was `3`  ordered-list-marker-value
```

The markdown-lint github action being used here says it is "with presets", but I couldn't quickly find what those preset were. `markdown-lint`'s default config is too picky.

Anyway, I realize I am well into overkill here. :) I'll approve this PR. It helps and this single instance isn't this time-important.

Reply:
This was the lint error:

https://github.com/open-telemetry/opentelemetry-js/actions/runs/9074537658/job/24933472224

```
./experimental/CHANGELOG.md:124:1 MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/1/1]
./experimental/CHANGELOG.md:125:1 MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 3; Style: 1/1/1]
```

Do you suggest converting it to `1. 1. 1.` instead of `* * *`?

Reply:
Nerd-me would see about disabling that `MD029/ol-prefix` check in the markdown-lint settings.  

Personally I think it is a misguided option. The idea is that the maintenance of writing ordered-lists in Markdown is reduced if all the bullets are `1.` because you don't need to re-number if you add or remove list items. But it comes at the cost of being able to unambiguously refer to "item N" when reading the source Markdown, instead of the rendered view.

Ah, it looks like this changed in more recent versions of markdownlint to default to `one_or_ordered`, so that either is fine by default: https://github.com/DavidAnson/markdownlint/issues/97

This got me digging a bit. It looks like the markdownlint-cli being used in `avto-dev/markdown-lint` GH action is about 4y old. That github action isn't being maintained.  As well, we have a `npm run lint:markdown` that behaves slightly differently than the markdown linting being done in the `lint.yml` GH workflow.

I'm going to open a separate PR as an alternative.
...
Opened here: https://github.com/open-telemetry/opentelemetry-js/pull/4713

Reply:
(Also, as an aside, my earlier comments showing usage of `npm install markdown-lint` could be misleading, because the repo is currently using `markdownlint-cli`, which uses `markdownlint`.  These are both unrelated to the `markdown-lint` that I used. Confusing.)